cmd/generate-workflow.go

```diff
@@ -29,6 +29,7 @@ func newGenerateWorkflowCmd() *cobra.Command {
 		Long: `This command will generate a Github workflow to build and deploy an application containerized 
 with draft on AKS. This command assumes the 'setup-gh' command has been run properly.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			flagValuesMap = make(map[string]string)
```

Comment:
is there a reason to have the `flagValuesMap` variable be at such a high scope?

Reply:
we already had it at this scope since this scope reads the workflowConfig to extract flags from the initial command run, i just added an initialization to be safe